doc/htmldoc/nest_sonata/nest_sonata_guide.rst

```diff
@@ -4,8 +4,26 @@ NEST SONATA guide
 =====================
 
 NEST supports building and simulating networks of point neurons described by the SONATA format [1]_. 
-This guide provides the details about how a SONATA network must be specified to be supported natively by NEST. 
+This guide provides the details about how a SONATA network must be specified to be supported natively by NEST.
 
+.. _sec:sonata_configure:
+
+Configure NEST for SONATA
+--------------------------
+
+To use SONATA with NEST, both `HDF5 <https://hdfgroup.org/>`_ and `h5py <https://www.h5py.org/>`_ must be installed on
+the system and NEST must be configured properly.
+
+In the installation of NEST, you need to add the following configuration option to
+your CMake.
+
+.. code-block:: sh
+
+    cmake -Dwith-hdf5=[ON </path/to/hdf5>]
```

Comment:
- I find this line misleading since it looks like a complete cmake invocation, but isn't. I would rather write that you need to add the `-Dwith-hdf5=ON` option to the cmake invocation. @jessica-mitchell What do you think?
- Does `/path/to/hdf5` really work?

Reply:
I agree, I think incomplete commands can be confusing.